### PR TITLE
feat(collector): apply restart API at in-progress status icon

### PR DESCRIPTION
**Pull request by skdud4659**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [ ] Not that difficult

### Type of Change
- [ ] New feature
- [ ] Bug fixes
- [x] Feature improvement
- [x] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
- apply detail link at job list icons 
- apply restart API at in-progress status icon
image

- changed minor css style by design QA
- add restart modal translation

### Things to Talk About
🚨 plz check by files changed

### CODE REVIEW

**Review by WANZARGEN — APPROVED** [review]
LGTM!

File: apps/web/src/services/asset-inventory/collector/collector-main/modules/modals/CollectorRestartModal.vue [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+const handleCloseModal = () => {
+    collectorPageStore.$patch({
+        visibleRestartModal: false,
+    });
+};
+
+/* API */
+const handleConfirm = async () => {
+    const collectorId = collectorPageStore.selectedCollector.collector_id;
+    await collectorPageStore.restartCollector(collectorId);
```
Comment (yuda110) [inline apps/web/src/services/asset-inventory/collector/collector-main/modules/modals/CollectorRestartModal.vue:34]: praise: simple!

**Review by yuda110 — APPROVED** [review]
LGTM!!!!

File: apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts [hunk 2]
```diff
@@ -92,12 +98,27 @@ export const useCollectorPageStore = defineStore('collector-page', {
...
                 const updatedCollectorIndex = this.collectors.findIndex((collector) => collector.collector_id === response.collector_id);
                 this.collectors[updatedCollectorIndex] = response;
+                showSuccessMessage(i18n.t('INVENTORY.COLLECTOR.ALT_S_UPDATE_SCHEDULE'), '');
                 return response;
             } catch (e) {
                 ErrorHandler.handleError(e);
                 throw e;
             }
         },
+        async restartCollector(collectorId: string) {
```
Comment (sulmoJ) [inline apps/web/src/services/asset-inventory/collector/collector-main/collector-page-store.ts:108]: nitpick: I think `startCollect` seems more appropriate.

**Review by sulmoJ — APPROVED** [review]
LGTM!

**Pull request merged**